flow/src/main/java/flow/KeyDispatcher.java

```diff
@@ -65,14 +69,30 @@ private KeyDispatcher(Activity activity, KeyChanger keyChanger) {
     // TODO(#126): this short-circuit may belong in Flow, since every Dispatcher we have implements it.
```

Comment:
Stop looking necessary in that KeyChanger would effectively replace Dispatcher, right?